src/wrap-store/wrapStore.js

```diff
@@ -80,5 +80,34 @@ export default (store, {
 
     // send initial state
     sendState();
+  };
+
+  /**
+   * Setup action handler
+   */
+  chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
```

Comment:
Can we simplify this as `chrome.runtime.onMessage.addListener(dispatchResponse)`